internal/wspace/finder.go

```diff
@@ -19,23 +19,39 @@ import (
 	"strings"
 )
 
-const workspaceFile = "WORKSPACE"
+var workspaceFiles = []string{"WORKSPACE.bazel", "WORKSPACE"}
 
-// Find searches from the given dir and up for the WORKSPACE file
+// FindWorkspaceFile returns a path to a file in the provided root directory,
+// either to an existing WORKSPACE or WORKSPACE.bazel file, or to root/WORKSPACE
+// by default. Note that this function does NOT recursively check parent directories
+func FindWorkspaceFile(root string) string {
+	for _, workspaceFile := range workspaceFiles {
+		path := filepath.Join(root, workspaceFile)
+		if _, err := os.Stat(path); err == nil {
+			return path
+		}
+	}
+	return filepath.Join(root, "WORKSPACE")
+}
+
+// FindRoot searches from the given dir and up for a directory containing a WORKSPACE file
 // returning the directory containing it, or an error if none found in the tree.
-func Find(dir string) (string, error) {
+func FindRoot(dir string) (string, error) {
```

Comment:
Let's call this `FindRepoRoot`. That term is used pretty consistently throughout the code base.